### PR TITLE
PN-3170: Composer Integration

### PR DESCRIPTION
**Pull request by Irina-Intuiface**
This branch fixes various problems that I encountered when integrating URLParser in Composer:

- Adding IA title and description to "ressources", where it's correctly interpreted
- Adding Descriptions to all decorators
- Adding dist to gitignore

### CODE REVIEW

File: libs/tools/schematics/interface-asset-schematics/src/create/files/package.json [hunk 1]
```diff
@@ -6,10 +6,12 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "npm run cleanDist && npx tsc --project tsconfig.ifd.json && shx cp ./src/package.json ./tmp/ && node --experimental-modules --es-module-specifier-resolution=node tmp/index_ifd.js && npm run cleanTmp && npx webpack --config webpack.config.js",
+    "build:composer": "npm run cleanDist && npx tsc --project tsconfig.ifd.json && shx cp ./src/package.json ./tmp/ && node --experimental-modules --es-module-specifier-resolution=node tmp/index_ifd.js && npm run cleanTmp && npx webpack --config webpack.config.js && npm run postBuild",
```
Comment (mikaelkurek) [inline libs/tools/schematics/interface-asset-schematics/src/create/files/package.json:9]: I'm not sure to understand why this command is used for ? Does everybody can use this command ?
Should we add documentation for this ?

**Review by mikaelkurek — COMMENTED** [review]
Except the new script, it's ok for me

File: libs/tools/schematics/interface-asset-schematics/src/create/files/package.json (hunk 1 above)
Comment (jviolas) [reply libs/tools/schematics/interface-asset-schematics/src/create/files/package.json:9]: Same comment. Specific script needed in our own CI for Composer integration should not be in the default template of an IA workspace.
Same for `postBuild` script

File: libs/tools/schematics/interface-asset-schematics/src/workspace-template/gitignore [hunk 2]
```diff
@@ -1 +1,8 @@
+# compiled output
+/dist
+
+# IDE - VSCode
+.vscode/*
```
Comment (jviolas) [inline libs/tools/schematics/interface-asset-schematics/src/workspace-template/gitignore:5]: VSCode settings can be commited.

File: libs/tools/schematics/interface-asset-schematics/src/create/files/package.json (hunk 1 above)
Comment (Irina-Intuiface) [reply libs/tools/schematics/interface-asset-schematics/src/create/files/package.json:9]: Indeed, this is only useful for us, when we're building the source code of an IA to integrate it in Composer. Having it here was just making sure that all future IA's will have easy access to these scripts, but it has no purpose for our customers. Any ideas of where I can put them instead?

Comment (Irina-Intuiface) [reply libs/tools/schematics/interface-asset-schematics/src/create/files/package.json]: I removed the scripts, we'll just copy what we need manually for further Typescript IAs

**Review by mikaelkurek — APPROVED** [review]

**Pull request merged**